src/devices/cpu/tms34010/34010ops.hxx

```diff
@@ -10,6 +10,10 @@
 ***************************************************************************/
 
 
+// for log2
+#include <cmath>
+// for popcount32, count_leading_zeroes
+#include "eminline.h"
```

Comment:
This should already be included via `emu.h`

Reply:
OK, dropped both includes.